integration_tests/deploy/Dockerfile-build

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 DSR Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+############################
+# build app
+############################
+FROM golang:bullseye
+
+WORKDIR /go/src/dc-ledger
+COPY app ./app/
+COPY cmd ./cmd/
+COPY testutil ./testutil/
+COPY utils ./utils/
+COPY x ./x/
+COPY go.mod go.sum Makefile ./
+COPY docs/docs.go ./docs/
+COPY docs/static ./docs/static/
+
+ARG DCL_VERSION
+ARG DCL_COMMIT
+RUN LEDGER_ENABLED=false make
+
+RUN go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@v1.0.0
```

Comment:
It looks the same as `Dockerfile-build` in the root expect the last line, doesn't it? Should this image depend on the root one then?

Reply:
No, `Dockerfile-build` and `integration_tests/deploy/Dockerfile-build` use different base images because the artifact built in the first one is intended for use on regular containers in docker environment (`Dockerfile`) while the artifacts built in the second one are intended for use on the Ubuntu container where the new node is deployed in the deployment test (`integration_tests/deploy/Dockerfile`). (The operating systems in `Dockerfile` and `integration_tests/deploy/Dockerfile` are not binary compatible with each other.)